libtenzir/builtins/aggregation-functions/distinct.cpp

```diff
@@ -109,6 +109,10 @@ class plugin : public virtual aggregation_function_plugin {
     };
     return caf::visit(f, input_type);
   }
+
+  auto aggregation_default() const -> data override {
+    return caf::none;
```

Comment:
Should this be an empty list?

Reply:
The problem is that you don't know the exact type here. The `series_builder` makes a `list<null>` out of the empty list. I'm not sure whether that's the best outcome, but I'll make this change anyway as it sounds reasonable.

Reply:
I agree that it is reasonable to have.